owncloud/test/config.sample.json

```diff
@@ -1,5 +1,5 @@
 {
-    "owncloudURL": "http://127.0.0.1/",
+    "owncloudURL": "http://127.0.0.1:8888/",
```

Comment:
why did you set the owncloud url to the same as the cypress url? I doubt this is working

Reply:
Cypress baseUrl is just the address he sends requests to. I set it manually here since I did not look into changing this on-the-fly. We could handle this by calling cypress.config('baseUrl', the.url.owncloud.core.runs.on).